library/src/main/java/com/github/windsekirun/rxsociallogin/kakao/KakaoLogin.kt

```diff
@@ -58,6 +58,12 @@ class KakaoLogin constructor(activity: androidx.fragment.app.FragmentActivity) :
 
     override fun login() {
         checkSession()
+
+        if (sessionCallback != null) {
```

Comment:
@greedy0110 LGTM, There may be some code improvements.

In L77, we have `removeCallback` logic in `onDestroy` methods, so it can be extract one methods

```
private fun removeSession() {
   if (sessionCallback != null) {
     Session.getCurrentSession().removeCallback(sessionCallback)
     sessionCallback = null
   }
```

However, it is not necessary, so please modify it at your discretion or let us know again.